samples/sample-ec2-with-codedeploy/scripts/install-deps.sh

```diff
@@ -34,7 +34,7 @@ echo "<VirtualHost *:80>
   ProxyRequests Off
   ProxyPass / http://localhost:8080/
   ProxyPassReverse / http://localhost:8080/
-</VirtualHost>" >> sudo /etc/httpd/conf/httpd.conf
+</VirtualHost>" /etc/httpd/conf/httpd.conf >>sudo
```

Comment:
This would not work I think, would it?

Reply:
That does look odd... Let me move this to draft and investigate

Reply:
I modified this as follows:

```sh
# Append VirtualHost configuration
echo "
<VirtualHost *:80>
  ProxyRequests Off
  ProxyPass / http://localhost:8080/
  ProxyPassReverse / http://localhost:8080/
</VirtualHost>
" | sudo tee -a /etc/httpd/conf/httpd.conf >/dev/null
```